src/common/models/modifier.ts

```diff
@@ -0,0 +1,36 @@
+
+export interface Modifiers {
+  [key: string]: Modifier;
+}
+
+export interface Modifier {
+  label: string;
+  total: number;
+  id: ModifierTypes;
+}
+
+export enum ModifierTypes{
+  STR,
+  DEX,
+  CON,
+  INT,
+  WIS,
+  CHA
+}
+
+export const modifierList: {label: string, type: ModifierTypes}[] = [
+  {label:"Strength", type: ModifierTypes.STR}, 
+  {label:"Dexterity", type: ModifierTypes.DEX}, 
+  {label:"Constitution", type: ModifierTypes.CON}, 
+  {label:"Intelligence", type: ModifierTypes.INT}, 
+  {label:"Wisdom", type: ModifierTypes.WIS}, 
+  {label:"Charisma", type: ModifierTypes.CHA}
+]
+
+export const formatModifierList = () => {
+  let newModifierList:Modifiers = {}
+  modifierList.forEach(mod =>{
+    newModifierList[mod.type] = {label: mod.label, total:0, id: mod.type}
+  })
```

Comment:
I can explain this better in person but what we are doing is hard coding enum types. So any time we want to reference a specific type we use the enum. Similar to what modifierList is doing. 

Here I use a loop to generate the default modifiers. I think this should be all of them so we shouldn't have to add additional ones.